.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Contributing
```

Comment:
Maybe add information about the CLA (Contributor License Agreement) that is required to contribute? (If we still require it, that is)

Reply:
That's right. But how could we describe it here? Could you state a suggestion here @ed255 ?